pallets/rmrk-core/src/functions.rs

```diff
@@ -509,11 +525,10 @@ where
 	/// Output:
 	/// - Removing a `child` from the Children StorageMap of the `parent`
 	pub fn remove_child(parent: (CollectionId, NftId), child: (CollectionId, NftId)) {
-		Children::<T>::mutate(parent, |v| {
-			*v = v.iter().filter(|&nft| *nft != child).cloned().collect();
-		});
+		Children::<T>::remove((parent.0, parent.1), (child.0, child.1));
 	}
 
+	/// I don't think we use this at all? [delete this statement if i'm wrong or this function if not]
```

Comment:
We don't use it, but it could be used as a helper function for RPC API or something in the future. Either choice should be okay.

Reply:
deleting, can always implement again later if we want to use it.

Reply:
[196c9d8](https://github.com/rmrk-team/rmrk-substrate/pull/95/commits/196c9d8165e37bc0e2c93d132f456a0f049ea5f4)